Readme_TimeInCapitals.md

```diff
@@ -0,0 +1,25 @@
+# TimeInCapitals
+
+>Simple API that allows you to get time for capitals
+
+## Table of contents
+* [General info] (#general-info)
+* [Technologies] (#technologies)
+* [Setup] (#setup)
+* [Features] (#features)
+
+## General info
+TimeInCapitals is a SpringBoot app.
+
+Simple API that displays the time in a specific capital.
+
+## Technologies
+* SpringBoot 2.1.7
+* JAVA 1.8
+
+## Setup
+Run TimeInCapitalsApplication.java
```

Comment:
Ten opis w żaden sposób nie wyjasnia, jak mam odpalić aplikację. O jaki "Run" chodzi?

Reply:
Poprawione, tylko nie wiem czy jakoś tak bardziej ogólnie nie powinienem tego przedstawić?

Reply:
Przygotuj skrypt, który wywoła odpowiednie polecenia i poinstruuj użytkownika, aby uruchomił ten skrypt.

Dodatkowo jeśli chcesz wykorzystać wrapper mavena, to pamiętaj żeby też się znalazł w repozytorium.

Reply:
dodałem plik start.sh